timestream.go

```diff
@@ -211,15 +227,26 @@ func (t TimeStreamAdapter) runReadRequestQuery(q *prompb.Query) (result prompb.Q
 		return
 	}
 
-	timeStreamQueryOutput, err := t.Query(&timestreamquery.QueryInput{
-		QueryString: &task.query,
-	})
+	var timeSeries []*prompb.TimeSeries
+	var innerErr error
+	err = t.QueryPages(
+		&timestreamquery.QueryInput{
+			QueryString: &task.query,
+		},
+		func(output *timestreamquery.QueryOutput, lastPage bool) (continueIterating bool) {
+			continueIterating = true
```

Comment:
I'm not sure if this function should always true.

Reply:
This is not very well documented, but my understanding is that it only needs to return false if it wants to exit before all pages of the response have been read. As far as I can tell this works, but setting `continueIterating = !lastPage` would make it nicer. I'll do that and see what happens.

Reply:
Setting `continueIterating = !lastPage` works fine!